net-analyzer/prelude-correlator/prelude-correlator-4.0.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit systemd distutils-r1
```

Comment:
Sort lexically whenever possible. Here ordering doesn't change anything, so it's possible.

Reply:
ok